### ISSUE TITLE
[2.4.5] External video "unmutable"

### ISSUE THREAD

**Issue by PhMemmel** [bigbluebutton/bigbluebutton#14618]
**Describe the bug**
When playing an external video a user can click on the speaker symbol in the appearing volume bar to mute the video. It is being muted, but instantly back to full volume again. Dragging the slider to zero works however.

**To Reproduce**
1. Open session, share external video
2. As viewer click on the speaker symbol of the video's volume bar
3. Video is being muted but volume is instantly restored back to full

**Expected behavior**
Video should be muted by clicking on the speaker symbol

**Actual behavior**
Just shortly muted, instantly restored back to full volume

**BBB version:**
2.4.5

**Desktop (please complete the following information):**
 - OS: Win10 latest
 - Browser: Chrome latest

**Comment by gabriellpr** [comment]
I tried to reproduce the error, but it didn't work on Google Chrome and Mozilla Firefox.
The mute is working normal here.

BBB version: most recent on 2.4
Ubuntu: 20.04.4 LTS
Firefox: 98.0.1
Chrome:  99.0.4844.74

**Comment by PhMemmel** [comment]
@gabriellpr Thank you for the feedback.

First of all just to make sure: You tried on viewer side to mute, right?

Apart from that: It doesn't seem to be a problem with youtube videos: With youtube videos everything works correctly.

However it does seem to be a problem for mp4 videos. I tried a sample from a public website (https://download.samplelib.com/mp4/sample-5s.mp4) and I also tried to share a mp4 video from my own nextcloud instance. In both cases I had the "volume back to full" issue.
